File: lib-mvpbase-appcompat/src/main/java/com/baozi/mvp/presenter/EmptyPresenter.java
package com.baozi.mvp.presenter;

import android.view.View;

/**
 * Created by baozi on 2017/12/20.
 */

public final class EmptyPresenter extends BasePresenter {

    @Override
    public void onCreate() {
    }

    @Override
    public void onRefreshData() {

    }

    @Override
    public View getContentView() {
        return null;
    }

    @Override
    public void cancelNetWork() {

    }

    @Override
    public void netWorkError(Throwable throwable) {

    }
}

File: lib-mvpbase-kotlin/src/main/java/com/linfeng/mvp/presenter/EmptyPresenter.kt
package com.linfeng.mvp.presenter

import android.view.View
import com.linfeng.mvp.view.BaseView
import com.linfeng.mvp.view.UIView

/**
 * Created by baozi on 2017/12/20.
 */

class EmptyPresenter : BasePresenter<UIView>() {

    override fun onCreate() {}

    override fun onRefreshData() {

    }

    override fun cancelRequest() {

    }

    override fun netWorkError(throwable: Throwable) {

    }
}
